Cognigy/WebchatWidget
# Improve keyboard a11y of time fields in datetime pickers

Improved the accessibility of DateTime pickers.

Success Criteria:
- The DateTime Pickers should be keyboard accessible

How to test:
- You can use this endpoint to test this PR: https://endpoint-dev-v4.cognigy.ai/599d6418ab91a6f455920996c53347e2a13d660ea1652707ab77bb0ff469a41b
- Test all the Date pickers and DateTime pickers by navigating them using 'TAB' and 'SHIFT +TAB' keys
Previously, you will not be able to navigate out of the time input field (keyboard trap). This should be fixed now
